Packages/tests/Basic/UTF_AnalysisBrowserTest.ipf

```diff
@@ -164,3 +164,82 @@ static Function LoadDependentStimsetsFromPXP()
 
 	KillWindow $abWin
 End
+
+static Function RoundTripNWBDepStimsets()
+
+	string wbPanel, fName, abWin, sweepBrowsers, stimsets, refList
+	string setNameFormula = "formula"
+	string setNameBase = "baseSet"
+	string baseFileName = "RoundTripNWBDepStimsets.nwb"
+	variable nwbVersion = GetNWBVersion()
+
+	DFREF dfr = GetWBSvdStimSetParamPath()
+	KillDataFolder dfr
+	wbPanel = WBP_CreateWaveBuilderPanel()
+	CHECK_PROPER_STR(wbPanel)
+	PGC_SetAndActivateControl(wbPanel, "SetVar_WaveBuilder_P0", val=10)
+	PGC_SetAndActivateControl(wbPanel, "SetVar_WaveBuilder_P2", val=1)
+	PGC_SetAndActivateControl(wbPanel, "setvar_WaveBuilder_baseName", str=setNameFormula)
+	PGC_SetAndActivateControl(wbPanel, "button_WaveBuilder_SaveSet")
+	PGC_SetAndActivateControl(wbPanel, "setvar_combine_formula_T6", str="A*2", switchTab=1)
+	PGC_SetAndActivateControl(wbPanel, "setvar_WaveBuilder_baseName", str=setNameBase)
+	PGC_SetAndActivateControl(wbPanel, "button_WaveBuilder_SaveSet")
+	KillWindow/Z wbPanel
```

Comment:
Please use the stimset creation API instead of the wavebuilder, examples are at https://github.com/AllenInstitute/MIES/blob/main/Packages/doc/ipf/example-stimulus-set-api.ipf.